presto-main/src/main/java/com/facebook/presto/operator/window/WindowAnnotationsParser.java

```diff
@@ -37,12 +38,15 @@ private WindowAnnotationsParser() {}
     {
         WindowFunctionSignature[] signatures = clazz.getAnnotationsByType(WindowFunctionSignature.class);
         checkArgument(signatures.length > 0, "Class is not annotated with @WindowFunctionSignature: %s", clazz.getName());
+        WindowFunctionOptions options = clazz.getAnnotation(WindowFunctionOptions.class);
+        boolean canIgnoreNulls = options != null && options.canIgnoreNulls();
         return Stream.of(signatures)
-                .map(signature -> parse(clazz, signature))
+                .map(signature -> parse(clazz, signature, canIgnoreNulls))
                 .collect(toImmutableList());
     }
 
-    private static SqlWindowFunction parse(Class<? extends WindowFunction> clazz, WindowFunctionSignature window)
+    private static SqlWindowFunction parse(Class<? extends WindowFunction> clazz, WindowFunctionSignature window,
+            boolean canIgnoreNulls)
```

Comment:
Please put all arguments in one line or each in separate line.

Reply:
Ok.